analysis/staticAnalysisReport.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "Eval Static Analysis"
+author: "Chakshu Goyal"
+date: "07/06/2018"
+output:
+  html_document: default
+  pdf_document: default
+---
+
+## 
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(ggplot2)
+library(tidyverse)
+library(tools)
+```
+
+## Load the data
+Usage of the function eval in R-packages
+```{r}
+evaldf <- read_csv("evals.csv")
+```
+
+## Plot occurences of evals in packages
+```{r one}
+# Occurences of evals in each package plot
+ggplot(evaldf , aes(pkg)) + geom_bar() + theme(axis.text.x = element_blank()) + labs(y="Number of evals", x="packages", title="Figurative distribution of the usage of eval in various packages")
+
+# No. of occurences of eval in each package table
+evalCntPerPkg <- plyr::count(evaldf , "pkg")
```

Comment:
1. this can be done using dplyr (it is better to stick to one of these libs):

```
dplyr::count(evaldf, pkg)
```

2. the _now_ used naming scheme is ala-python `lowercase_separated_by_underscores`

Reply:
Yes, okay! Acknowledged.